tests/calc/test_thermo.py

```diff
@@ -665,8 +666,8 @@ def test_sensitive_sounding():
     assert_almost_equal(lfc_temp, 20.498 * units.degC, 2)
 
     pos, neg = surface_based_cape_cin(p, t, td)
-    assert_almost_equal(pos, 0.1115 * units('J/kg'), 3)
-    assert_almost_equal(neg, -6.0866 * units('J/kg'), 3)
+    assert_almost_equal(pos, 0.0 * units('J/kg'), 3)
+    assert_almost_equal(neg, 0.0 * units('J/kg'), 3)
```

Comment:
While correct, I think this means the test isn't testing what we want any more. Let me see if I can nudge these values to keep the behavior we want to test.

Reply:
Ok, well it looks like even reverting back to the implementation of `surface_based_cape_cin` to before #902 was fixed isn't giving the error (could be any of a number of other things that changed that allows it to pass). So I say let's just tweak the sounding to some values that give some small positive area to at least try to test something useful here:
```python
    t = units.Quantity([25.1, 24.5, 20.2, 21.6, 21.4, 20.4, 20.2, 14.4, 13.2, 13., 6.8, -3.3,
                        -13.1, -13.7, -17.9, -25.5, -26.9, -37.9, -46.7, -48.7, -52.1, -58.9],
                       'degC')
```

Reply:
This resolved the 0/0 issue 😄